spec.md

```diff
@@ -112,6 +113,43 @@ This header allows brokers to reject requests from marketplaces for versions the
 The marketplace MUST authenticate with the service broker using HTTP
 basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
 
+## Originating Identity
+
+Often a service broker will need to know the identity of the user that
+initiated the request from the platform. For example, this might be needed
+for auditing or authorization purposes. In order to facilitate this, the
+platform will need to provide this identification information to the broker
+on each request. If the platform wishes to provide this information then
+it MUST include an additional HTTP header on all requests it sends to the
+broker, except the request to retrieve the catalog since that is not
+associated with a platform's user. The format of the header MUST be:
```

Comment:
1. perhaps but if a query for the latest catalog is a cron job who would be the user in that case? The same person who registered it originally? What if they've been deleted?
2. probably true - I added some text, see what you think.
3. "wishing" vs "if" is the same thing to me, so I don't mind changing that part if people want.  But, we do need a MUST in there so that people know they need to adhere to what we're defining if they're going to support this optional feature.  "CAN" isn't an RFC2119 keyword.